param/__init__.py

```diff
@@ -750,46 +750,37 @@ class Number(Dynamic):
 
     """
 
-    __slots__ = ['bounds','_softbounds','inclusive_bounds','set_hook', 'step']
+    __slots__ = ['bounds', 'softbounds', 'inclusive_bounds', 'set_hook', 'step']
```

Comment:
Getting rid of the underscore makes sense as the properties didn't really do anything, but couldn't this have backward compatibility implications?

Reply:
I suppose stateful things like pickling shouldn't touch slots directly and the API is the same as it was when properties were used. It *should* be ok but I am always wary when messing with slots in param.